daprdocs/content/en/operations/support/alpha-apis.md

```diff
@@ -0,0 +1,16 @@
+---
+type: docs
+title: "Alpha APIs"
+linkTitle: "Alpha APIs"
+weight: 5000
+description: "List of current alpha APIs"
+---
+
+| Building block/API | gRPC | HTTP | Description | Documentation | Version introduced | 
+| ------------------ | ---- | ---- | ----------- | ------------- | ------------------ |
+| Query State    | [Query State proto](https://github.com/dapr/dapr/blob/5aba3c9aa4ea9b3f388df125f9c66495b43c5c9e/dapr/proto/runtime/v1/dapr.proto#L44)     | `v1.0-alpha1/state/statestore/query` | The state query API enables you to retrieve, filter, and sort the key/value data stored in state store components. | [Query State API]({{< ref "howto-state-query-api.md" >}}) | v1.5 |
```

Comment:
can each of these be tied to the latest runtime version? 

Now that I think about it it is going to be something that is going to be a moving target potentially.